web-marketplace/src/pages/Media/Media.tsx

```diff
@@ -22,10 +22,20 @@ const Media = (): JSX.Element => {
       anchored: false,
     });
 
+  // Handle old metadata schema
+  const previewPhoto = metadata?.['regen:previewPhoto'];
+  const isPreviewPhotoString = typeof previewPhoto === 'string';
+  const previewPhotoUrl = isPreviewPhotoString
+    ? previewPhoto
+    : previewPhoto?.['schema:url'] ?? '';
+  const previewPhotoCredit = isPreviewPhotoString
+    ? ''
+    : previewPhoto?.['schema:creditText'] ?? '';
```

Comment:
I believe we should not really support that anymore. All projects on prod should have been migrated to the latest format. But that might not be the case for all projects on staging indeed. In this case, that should be updated manually from the db rather that supported from the UI.
This brings up the need again to reset staging data (both db and redwood testnet).

Reply:
Opened an issue for further discussion: https://github.com/regen-network/rnd-dev-team/issues/1784

Reply:
I can remove that logic then but yes that's the case for most of our projects on staging.